README.md

```diff
@@ -90,6 +90,8 @@ Currently the kubemark actuator allows to configure the following test scenarios
 
 Other configuration options:
 - `deletionTimeout` - how much time to wait before a machine gets deleted from the cluster after setting machine deletion timestamp
+- `numCores` - for a number of cores a kubemark node will claim to have
+- `memoryCapacity` - for a memory a kubemark node will claim to have
```

Comment:
"will claim to have" - what will it actually have?

Reply:
Claim is it will pretend. It's still a fake node so the memory capacity is actually faked. Maybe `to claim` is not the right word here. Maybe `memory a kubemark node will report`?